### ISSUE TITLE
Ability to use current package.json version or tag?

### ISSUE THREAD

**Issue by mcdermed** [webpro/release-it#195]
It appears there's no way to use the tool without bumping the current version or tag. Say for instance I've already run `npm version minor` and pushed the tag into my CI environment. When it attempts to upload the release, I get the error message described in https://github.com/webpro/release-it/issues/172.

```
$ release-it
✔ Init (release 0.16.1 for xxx)
✔ Bump version
✖ GitHub release
ERROR 404 Not Found: Not Found ()
```

```
release-it --no-increment
ERROR increment.split is not a function

release-it increment=""
ERROR Unable to find an existing Git tag, or package.json#version. No or invalid version provided.
```
- Ideally we could tell the tool to skip increment and just validate the current tag semver

package.json
```
{
  "version": "0.16.0",
  ...
}
```
.release-it.json
```
{
    "non-interactive": true,
    "src": {
        "commit": false,
        "tag": false,
        "push": false
    },
    "npm": {
        "publish": false
    },
    "github": {
        "release": true,
        "assets": "xxx.xxx"
    }
}
```

If this sounds like a valid use case for the tool I'd love to open a PR with these changes. Thanks!

**Comment by webpro** [comment]
Thanks, @mcdermed! Yes, I think `--no-increment` is indeed a valid use case.

**Comment by webpro** [comment]
Released in [6.0.0-rc.0](https://github.com/webpro/release-it/releases/tag/6.0.0-rc.0). You can install with `npm install release-it@rc`.
